### PR TITLE
Change default monitoring port

### PR DESCRIPTION
**Pull request by ericvn**
Update the default monitoring port from 9093 to 15014.

This is discussed in https://github.com/istio/istio/issues/11125

Per @linsun, I am opening against 1.1.

### CODE REVIEW

**Comment by linsun** [comment]
lgtm - probably need a sweep of istio.io as well on the new port.

@costinm @mandarjog PTAL this is per our discussion yesterday

cc @wenchenglu

**Comment by linsun** [comment]
ran through this port tool:  https://www.wintelguy.com/port-search/. 15014 comes out clear :)

**Comment by ericvn** [comment]
/retest

Added another commit

File: install/kubernetes/helm/istio-remote/values.yaml [hunk 1]
```diff
@@ -34,6 +34,9 @@ global:
   # Default tag for Istio images.
   tag: master-latest-daily
 
+  # monitoring port used by mixer, pilot, galley
```
Comment (sdake) [inline install/kubernetes/helm/istio-remote/values.yaml]: we should hardcode the port.  We do that for the rest of Istio.

Comment (sdake) [reply install/kubernetes/helm/istio-remote/values.yaml]: (there was great debate about this in the past as well, and we decided on the hardcodes).

Comment (ericvn) [reply install/kubernetes/helm/istio-remote/values.yaml]: I removed using a value for the monitoring port in istio-remote that I had added.
I left the value in the istio branch (just changed the value).

File: pilot/pkg/proxy/envoy/v2/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # Debug interface
 
-The debug handlers are configured on the monitoring port (default 9093) as well
+The debug handlers are configured on the monitoring port (default 15014) as well
```
[the remark this answers is not in the document]
Comment (ericvn) [reply pilot/pkg/proxy/envoy/v2/README.md]: Will work on an istio.io PR shortly.

[the remark this answers is not in the document]
Comment (ericvn) [reply pilot/pkg/proxy/envoy/v2/README.md]: Two istio.io PRs submitted, one for English, other for translated files.

**Comment by GregHanson** [comment]
cc @wenchenglu

**Pull request merged**